ivy/functional/backends/torch/elementwise.py

```diff
@@ -562,9 +565,12 @@ def subtract(
     x2: Union[float, torch.Tensor],
     /,
     *,
+    alpha: Optional[Union[int, float]] = None,
     out: Optional[torch.Tensor] = None,
 ) -> torch.Tensor:
     x1, x2 = ivy.promote_types_of_inputs(x1, x2)
+    if alpha:
```

Comment:
`if alpha not in (1, None)`

we cannot do `if alpha` as `alpha=0` will boolean evaluate to `False`, for example.